### DISCUSSION TITLE
new Quarto book - would be great to see this in the gallery

### DISCUSSION THREAD

**Discussion by davidfoxcroft** [quarto-dev/quarto-cli#3022] [Show and Tell]
Hi, I've converted a book from bookdown to quarto and it's working well! It would be great to see [this](https://davidfoxcroft.github.io/lsj-book/) in the quarto gallery. 

The only glitch I can't fix / workaround is in the epub version, where math in the footnote pop up isn't rendered properly, though it is in the main view. But this is an issue with Calibre rather than Quarto.

**Comment by mcanouil** [comment]
Out of curiosity, do you use any custom layout or anything out of the standard book format?

**Reply by davidfoxcroft** [reply]
No, it's all standard layout. 

I had a bit of a challenge figuring out the math syntax that would work for both html and latex (pdf), especially in the tables (rendered using the R huxtable package).

**Reply by mcanouil** [reply]
I can easily imagine, there are several discussions/issues on the math equation between HTML and PDF/LaTeX. 😅

**Reply by jav-ed** [reply]
The book looks very nice, especially the zoom effect when hovering over images

**Comment by cderv** [comment]
@davidfoxcroft you can do a PR in quarto-web, like this [one](https://github.com/quarto-dev/quarto-web/pull/548), this [one](https://github.com/quarto-dev/quarto-web/pull/646) or this other [one](https://github.com/quarto-dev/quarto-web/pull/595)

Thanks !

**Comment by christofs** [comment]
This looks brilliant, thanks for sharing. I love the zoom on the figures, and the chapter-level pointers to edit the page or open an issue.
